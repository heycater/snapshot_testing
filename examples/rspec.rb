require "bundler/setup"
require "rspec/autorun"
require "snapshot_testing/rspec"

RSpec.configure do |config|
  config.include SnapshotTesting::RSpec
end

RSpec.describe "Example" do
  it "takes a snapshot" do
    expect("hello").to match_snapshot
    expect("goodbye").to match_snapshot
  end

  it "takes a named snapshot" do
    expect("named").to match_snapshot("named.rspec.txt")
  end
end
