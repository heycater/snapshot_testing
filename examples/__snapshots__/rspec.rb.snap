snapshots["takes a snapshot 1"] = <<-SNAP
hello
SNAP

snapshots["takes a snapshot 2"] = <<-SNAP
goodbye
SNAP
